packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -655,7 +655,7 @@ Future<Map<String, dynamic>> _deviceToMap(Device device) async {
     'id': device.id,
     'name': device.name,
     'platform': getNameForTargetPlatform(await device.targetPlatform),
-    'emulator': device.isLocalEmulator
+    'emulator': await device.isLocalEmulator,
```

Comment:
@Hixie so that it's natively encodable into JSON, which is apparently how we communicate info to our IntelliJ plugin